packages/webview_flutter/webview_flutter_android/android/src/main/java/io/flutter/plugins/webviewflutter/InstanceManager.java

```diff
@@ -60,18 +56,17 @@ public interface FinalizationListener {
   private final FinalizationListener finalizationListener;
 
   private long nextIdentifier = MIN_HOST_CREATED_IDENTIFIER;
-  private boolean isClosed = false;
+  private boolean hasFinalizationListenerStopped = false;
 
   /**
    * Instantiate a new manager.
    *
-   * <p>When the manager is no longer needed, {@link #close()} must be called.
+   * <p>When the manager is no longer needed, {@link #stopFinalizationListener()} must be called.
    *
    * @param finalizationListener the listener for garbage collected weak references.
    * @return a new `InstanceManager`.
    */
-  @NonNull
-  public static InstanceManager open(@NonNull FinalizationListener finalizationListener) {
+  public static InstanceManager create(@NonNull FinalizationListener finalizationListener) {
```

Comment:
Did you mean to remove non null from the create call?

Reply:
Accident removal after updating to for the lint warnings. Added it back.